keras_nlp/models/bart/bart_preprocessor.py

```diff
@@ -160,20 +162,26 @@ def __init__(
             truncate=truncate,
             sequence_length=encoder_sequence_length,
         )
+        # The decoder is packed a bit differently; the format is as follows:
+        # `[end_token_id, start_token_id, decoder_input_ids, end_token_id, pad_token_ids]`.
```

Comment:
nit: this line length looks over 80?

 Maybe `[end_token_id, start_token_id, ...tokens..., end_token_id, ...padding...]` I would actually fine that a little more readable anyway, and gets you in limit.